posts/discriminative-vs-generative/index.qmd

```diff
@@ -0,0 +1,601 @@
+---
+title: "Generative vs. discriminative models"
+subtitle: "Thinking through the differences between generative and discriminative models."
+date: "2022/05/29"
+image: "graph.png"
+categories: [ML, theory, factor graphs, PGMs]
+echo: false
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .qmd
+      format_name: quarto
+      format_version: '1.0'
+      jupytext_version: 1.13.7
+  kernelspec:
+    display_name: Python 3.6.15 ('fgm')
+    language: python
+    name: python3
+---
+
+```{python}
+#| eval: false
+!pip install -r requirements.txt
+```
+
+```{python}
+from graphviz import Source
+import matplotlib.pyplot as plt
+```
+
+```{python}
+plt.rcParams['text.usetex'] = True
+```
+
+Generative and discriminative models are known to have complementary strenghts.
+A lot has been written comparing these two paradigms, sometimes more clearly than others.
```

Comment:
This reads like an Escher sentence. Think if we want to keep it like this it should be:
"A lot has been written comparing these two paradigms, some more clear than others." 
I still think it reads unnaturally however and would push towards a re-write. Happy to leave it if you like it though.

Reply:
Given we also don't agree on what the subject of the sentence is - the writing or the written - i've cut it out completely.